### PR TITLE
Fix issues of postgres not properly populating foreign key on photos table with net new seeding script.

### PR DESCRIPTION
**Pull request by christulin**

### CODE REVIEW

File: database/Models/index.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 
 const sequelize = new Sequelize({
   dialect: 'postgres',
-  user: 'christophertulin',
+  user: process.env.POSTGRES_USER,
   database: 'sdc',
-  password: 'test',
-  port: 5432,
-  host: 'localhost',
+  password: process.env.POSTGRES_PASSWORD,
```
Comment (amosNash) [inline database/Models/index.js:9]: Good use of env variables.

File: database/Models/index.js [hunk 2]
```diff
@@ -1,13 +1,14 @@
 const { Sequelize } = require('sequelize');
 const { applyAdditionalSetup } = require('./additional-setup');
+const dotenv = require('dotenv');
```
Comment (amosNash) [inline database/Models/index.js:3]: Is this needed to get reference to your env variables?

**Review by amosNash — COMMENTED** [review]
Looks good! I have taken a look at this code a few times and I don't feel like there are any changes that are needed. Good work.

File: database/Models/index.js (hunk 2 above)
Comment (christulin) [reply database/Models/index.js:3]: Yes, there are other ways.. but the `dotenv` package makes it very straightforward and configurable

**Pull request merged**